megamek/src/megamek/common/QuirksHandler.java

```diff
@@ -228,14 +227,11 @@ public static String replaceUnitType(String unitId, String newUnitType) {
     }
 
     private static Map<String, List<QuirkEntry>> loadQuirksFile(String path) throws IOException {
-        final String METHOD_NAME = "loadQuirksFile(String)";
-        
         Map<String, List<QuirkEntry>> quirkMap = new HashMap<>();
 
         File file = new File(path);
         if (!file.exists() || !file.isFile()) {
-            getLogger().log(QuirksHandler.class, METHOD_NAME, LogLevel.WARNING,
-                            "Could not load quirks from " + path);
+            getLogger().warning("Could not load quirks from " + path);
```

Comment:
Keeping this using the DefaultMMLogger?

Reply:
This has a setlogger method. I didn't want to just remove it.

```
// Use to pass in a fake logger for unit tests.
    static void setLogger(final MMLogger newLogger) {
        logger = newLogger;
    }
```